Snowflake.Data.Tests/IntegrationTests/SFDbDataReaderIT.cs

```diff
@@ -371,12 +371,12 @@ public void TestGetTimeSpanError()
                 TimeSpan timeSpanTime = ((SnowflakeDbDataReader)reader).GetTimeSpan(13);
 
                 reader.Close();
-                
+
                 CloseConnection(conn);
             }
         }
 
-        private void TestGetDateAndOrTime(string inputTimeStr, int? precision, SFDataType dataType)
+        private void TestGetDateAndOrTime(string inputTimeStr, int? precision, SFDataType dataType, DateTimeKind expectedKind)
```

Comment:
I see that in all executions expectedKind is unspecified. So this parameter is not really needed, we could remove it.

Reply:
done